cmd/juno/juno.go

```diff
@@ -176,6 +179,8 @@ const (
 	corsEnableUsage             = "Enable CORS on RPC endpoints"
 	versionedConstantsFileUsage = "Use custom versioned constants from provided file"
 	pluginPathUsage             = "Path to the plugin .so file"
+	logHostUsage                = "The interface on which the log level HTTP server will listen for requests."
```

Comment:
Should we add the logUsage flag to keep it consistent with the rest of the configuration? Like metricsUsage, grpcUsage, or httpUsage?